esx_service/vmdk_ops.py

```diff
@@ -736,14 +738,12 @@ def reset_vol_meta(vmdk_path):
     if not vol_meta:
        vol_meta = {}
     logging.debug("Reseting meta-data for disk=%s", vmdk_path)
-    if set(vol_meta.keys()) & {kv.STATUS, kv.ATTACHED_VM_UUID, kv.ATTACHED_VM_NAME}:
-          logging.debug("Old meta-data for %s was (status=%s VM name=%s uuid=%s)",
+    if set(vol_meta.keys()) & {kv.STATUS, kv.ATTACHED_VM_UUID}:
+          logging.debug("Old meta-data for %s was (status=%s VM uuid=%s)",
```

Comment:
And we should get it back. It is mutable but fairly stable and admin id VM by name. Why it is not there anymore ?